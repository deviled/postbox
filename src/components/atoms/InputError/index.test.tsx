import React from 'react';
import { render } from '__tests__/utils';
import { InputError } from '.';

describe('InputError.tsx', () => {
  it('should match snapshot', () => {
    const { baseElement } = render(<InputError>Unexpected error.</InputError>);

    expect(baseElement).toMatchSnapshot();
  });
});
