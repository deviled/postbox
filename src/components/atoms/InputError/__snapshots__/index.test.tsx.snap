// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputError.tsx should match snapshot 1`] = `
.c0 {
  font-size: 0.75rem;
  color: darkred;
}

<body>
  <div>
    <div
      class="c0"
    >
      Unexpected error.
    </div>
  </div>
</body>
`;
